tests/lib/managers/folders-test.js

```diff
@@ -276,4 +276,20 @@ describe('Folders', function() {
 		});
 	});
 
+	describe('deletePermanently()', function() {
+
+		it('should call BoxClient defaultResponseHandler method with the callback when response is returned', function(done) {
+			sandbox.mock(boxClientFake).expects('defaultResponseHandler').returns(done);
```

Comment:
That is what I wanted to ask , many tests are using withArgs('/folders/1234/metadata/global/properties... Why is that? Wouldn't it make sense only if you want to test with different arguments and test different possible responses?